### PR TITLE
Nebharg/telemetry token cache serialized

### PR DESCRIPTION
**Pull request by neha-bhargava**
#2185

### CODE REVIEW

File: src/client/Microsoft.Identity.Client/ITokenCacheInternal.cs [hunk 1]
```diff
@@ -75,6 +75,8 @@ internal interface ITokenCacheInternal : ITokenCache, ITokenCacheSerializer
         /// </summary>
         bool HasTokensNoLocks();
 
+        bool IsTokenCacheSerialized();
```
Comment (bgavrilMS) [inline src/client/Microsoft.Identity.Client/ITokenCacheInternal.cs:78]: CC @pmaytak as you're also planning to add a similar method. Happy merging :)

**Review by bgavrilMS — APPROVED** [review]

**Review by pmaytak — APPROVED** [review]

File: tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/UsernamePasswordIntegrationTests.cs [hunk 2]
```diff
@@ -29,8 +29,8 @@ public class UsernamePasswordIntegrationTests
 
         // Http Telemetry Constants
         private static Guid CorrelationId = new Guid("ad8c894a-557f-48c0-b045-c129590c344e");
-        private const string XClientCurrentTelemetryROPC = "2|1003,0|";
-        private const string XClientCurrentTelemetryROPCFailure = "2|1003,0|";
+        private const string XClientCurrentTelemetryROPC = "2|1003,0|0";
```
Comment (jennyf19) [inline tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/UsernamePasswordIntegrationTests.cs:32]: won't adding the new value first break pre-existing queries?

Comment (bgavrilMS) [reply tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/UsernamePasswordIntegrationTests.cs:32]: Good point, haven't caught that. New values should go at the end.

Comment (neha-bhargava) [reply tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/UsernamePasswordIntegrationTests.cs:32]: There are no existing queries for regional data currently. But good point to add new fields to end. I will create a PR to fix it. Thanks

**Pull request merged**